support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -49,6 +51,20 @@ export function filterBenefitByRegion(
 	return true;
 }
 
+export function filterBenefitByABTest(
```

Comment:
Very nice! This would be helpful for the newspaper benefits because atm they're bodged in by spreading in the whole product catalog and adding them in